packages/core/src/components/breadcrumb/breadcrumb.tsx

```diff
@@ -89,7 +91,7 @@ export class Breadcrumb {
     });
 
     this.mutationObserver.observe(
-      this.hostElement.querySelector('.crumb-items'),
+      this.hostElement.shadowRoot.querySelector('.crumb-items'),
```

Comment:
Need to observe `this.hostElement` because otherwise `ix-breadcrumb` will not render item changes e.g:

```tsx
  @State() items = Array.from({ length: 5 }).map((_, index) => `Item ${index}`);

  render() {
    return (
      <Host>
        <ix-button
          onClick={() => {
            this.items = [
              ...this.items,
              ...Array.from({ length: 3 }).map(
                (_, index) => `Added - Item ${index}`
              ),
            ];
          }}
        >
          Update Array
        </ix-button>
        <ix-breadcrumb visibleItemCount={3}>
          {this.items.map((item) => {
            return <ix-breadcrumb-item label={item}></ix-breadcrumb-item>;
          })}
        </ix-breadcrumb>
      </Host>
    );
  }
```